src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -446,6 +446,8 @@ public enum ConfigNodes {
 			"",
 			"",
 			"# This section is applied to new worlds as default settings when new worlds are detected."),
+	        "# You can adjust these settings for existing worlds using /townyworld ?"),

```

Comment:
This is already explained on line 303, but if we're going to go extra-redundant we need to add your new line to each new world setting that is modifiable in game.